packages/mdc-chips/chip/foundation.ts

```diff
@@ -110,7 +110,18 @@ export class MDCChipFoundation extends MDCFoundation<MDCChipAdapter> {
         // when adding it to the root client rect's width.
         const checkmarkWidth = checkmarkRect.height;
         const width = rootRect.width + checkmarkWidth;
-        return {...rootRect, width, height};
+        // Spread expression translate to Object.assign in Typescript.
```

Comment:
Good catch!

You can directly return the object with calculated height & width, so we don't need to use spread operator anymore.

```
return rect = {
  bottom: rootRect.bottom,
  height: rootRect.height,
  left: rootRect.left,
  right: rootRect.right,
  top: rootRect.top,
  width: width,
};
```

Reply:
Is there any special meaning this syntax? ```return rect = {};```
I guess you wanna talk this: ```return {}```
So I reflect it :)

Reply:
Sorry for the typo. Yes I meant `return {}`.